packages/smooth_ui_library/lib/widgets/smooth_sticky_list_view.dart

```diff
@@ -1,25 +1,24 @@
 import 'package:flutter/material.dart';
-import 'package:meta/meta.dart';
 
 typedef HasSameHeader = bool Function(int a, int b);
 
 class SmoothStickyListView extends StatefulWidget {
-  const SmoothStickyListView({
-    Key key,
-    this.itemCount,
-    @required this.itemExtend,
-    @required this.headerBuilder,
-    @required this.itemBuilder,
-    @required this.hasSameHeader,
-    this.padding,
-    this.headerPadding
-  }) : super(key: key);
+  const SmoothStickyListView(
+      {Key? key,
+      this.itemCount,
+      required this.itemExtend,
+      required this.headerBuilder,
+      required this.itemBuilder,
+      required this.hasSameHeader,
+      this.padding,
+      this.headerPadding})
+      : super(key: key);
 
-  final int itemCount;
+  final int? itemCount;
```

Comment:
I think we should prevent developers from putting `null` for `itemCount` => `final int itemCount;`